style.css

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
- [ ] Some styelint linter checks are failing.

**Stylelint errors:**
![image](https://user-images.githubusercontent.com/12745474/124605263-a2d61600-de63-11eb-922f-886a33e1278a.png)

To ensure a cleaner code kindly correct all styelint errors.